app/uk/gov/hmrc/pensionschemereturn/connectors/SchemeDetailsConnector.scala

```diff
@@ -47,6 +47,7 @@ class SchemeDetailsConnector @Inject() (appConfig: AppConfig, http: HttpClientV2
           "Content-Type" -> "application/json"
         )
       )
+      .transform(_.withRequestTimeout(appConfig.ifsTimeout))
```

Comment:
Do you think we need this here too? This connector doesn't go to IF (or at least not directly)